### PR TITLE
feat: icrc2_approve example call to dapp and demo

### PR DESCRIPTION
**Pull request by chorniy8pavlo**

### CODE REVIEW

File: examples/react-dapp/src/data/icrc49_call_canister.tsx [hunk 1]
```diff
@@ -54,6 +64,10 @@ export const icrc49CallCanisterSection: ISection = {
       title: "Ledger transfer",
       value: JSON.stringify(ledgerRequest, null, 2),
     },
+    {
+      title: "ICRC2 approve",
```
Comment (NadieinNFID) [inline examples/react-dapp/src/data/icrc49_call_canister.tsx:68]: ICRC-2 approve

File: examples/react-dapp/src/data/icrc49_call_canister.tsx [hunk 2]
```diff
@@ -54,6 +64,10 @@ export const icrc49CallCanisterSection: ISection = {
       title: "Ledger transfer",
```
Comment (NadieinNFID) [inline examples/react-dapp/src/data/icrc49_call_canister.tsx:64]: ICP transfer

**Review by dmitriiIdentityLabs — DISMISSED** [review]

File: examples/react-dapp/src/data/icrc49_call_canister.tsx [hunk 3]
```diff
@@ -54,6 +64,10 @@ export const icrc49CallCanisterSection: ISection = {
...
       value: JSON.stringify(ledgerRequest, null, 2),
     },
+    {
+      title: "ICRC2 approve",
+      value: JSON.stringify(icrc2approve, null, 2),
+    },
   ],
```
Comment (NadieinNFID) [inline examples/react-dapp/src/data/icrc49_call_canister.tsx:71]: you missed the ICRC-2 transfer example
https://github.com/dfinity/ICRC-1/tree/main/standards/ICRC-2#icrc2_transfer_from

File: examples/react-demo/src/data/icrc49_call_canister.tsx [hunk 4]
```diff
@@ -54,6 +64,10 @@ export const icrc49CallCanisterSection: ISection = {
       title: "Ledger transfer",
       value: JSON.stringify(ledgerRequest, null, 2),
     },
+    {
+      title: "ICRC2 approve",
```
Comment (NadieinNFID) [inline examples/react-demo/src/data/icrc49_call_canister.tsx:68]: ICRC-2 approve

File: examples/react-demo/src/data/icrc49_call_canister.tsx [hunk 5]
```diff
@@ -54,6 +64,10 @@ export const icrc49CallCanisterSection: ISection = {
       title: "Ledger transfer",
```
Comment (NadieinNFID) [inline examples/react-demo/src/data/icrc49_call_canister.tsx:64]: ICP transfer

File: examples/react-demo/src/data/icrc49_call_canister.tsx [hunk 6]
```diff
@@ -54,6 +64,10 @@ export const icrc49CallCanisterSection: ISection = {
...
       value: JSON.stringify(ledgerRequest, null, 2),
     },
+    {
+      title: "ICRC2 approve",
+      value: JSON.stringify(icrc2approve, null, 2),
+    },
   ],
```
Comment (NadieinNFID) [inline examples/react-demo/src/data/icrc49_call_canister.tsx:71]: missed ICRC-2 transfer here, as well

**Review by olekid — APPROVED** [review]

**Pull request merged**